modules/indexer/code/elastic_search.go

```diff
@@ -330,8 +330,13 @@ func extractAggs(searchResult *elastic.SearchResult) []*SearchResultLanguages {
 }
 
 // Search searches for codes and language stats by given conditions.
-func (b *ElasticSearchIndexer) Search(repoIDs []int64, language, keyword string, page, pageSize int) (int64, []*SearchResult, []*SearchResultLanguages, error) {
-	kwQuery := elastic.NewMultiMatchQuery(keyword, "content")
+func (b *ElasticSearchIndexer) Search(repoIDs []int64, language, keyword string, page, pageSize int, isMatch bool) (int64, []*SearchResult, []*SearchResultLanguages, error) {
+	searchType := "best_fields"
+	if isMatch {
+		searchType = "phrase_prefix"
+	}
```

Comment:
Are there constants for these somewhere? If not in the library, could we perhaps make some for our own reference?  
Might make it easier if/when this ever gets changed again.

Reply:
They are strings in the elastic library. If we put the constants in the  `elastic_search.go`, since they will only be referenced by elastic?

Reply:
That should be fine. I wish they had constants in the library itself, but this is the next best thing imo

Adding a reference here in case it ever comes up again: https://www.elastic.co/guide/en/elasticsearch/reference/7.0/query-dsl-multi-match-query.html#multi-match-types

Reply:
Thanks, 26623ec committed